PartyIcons/View/NameplateView.cs

```diff
@@ -26,6 +27,8 @@ public sealed class NameplateView : IDisposable
         private readonly PartyListHUDView _partyListHudView;
 
         private readonly IconSet _iconSet;
+        private const string _iconPrefix = "   ";
+        private readonly int[] ignorables = { 061521, 061522, 061523, 061540, 061542, 061543, 061544, 061547 };
```

Comment:
Private field names should start with underscore. Also too unspecific, should be something along the lines of '_ignoredIconIds'.